### DISCUSSION TITLE
No Configuration Works for text-prepender with sub-directoies

### DISCUSSION THREAD

**Discussion by balq60** [community/community#39791] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have a directory structure like:

root/IaC

In IaC is where the code exists that I would like to pre-pend text

The files in the root ARE getting updated. 

Just not in IaC

What is the correct configuration?

I have tried numerous.

- repo: https://github.com/dontirun/text-prepender
  rev: v0.1.0
  hooks:
    - id: text-prepender
      args:
        - '--text-file'
        - 'NOTICE'
        - '--path'
        - 'IaC'

That is my latest attempt.

**Comment by ossanna16** [comment]
Hi @balq60,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on text-prepender. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
